src/components/Home.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getStocks } from 'redux/stock/stocksSlice';
+import styles from 'styles/Home.module.css';
+import Hero from './Hero';
+import StockItem from './StockItem';
+
+const Home = () => {
+  const { stocks, hero } = useSelector((store) => store.stock);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getStocks());
+  }, [dispatch]);
+
+  return (
+    <div className="container">
+      <Hero data={hero} />
+      <ul className={styles.items}>
+        {
+          stocks.map((stock) => <StockItem key={stock.symbol} data={stock} />)
+        }
+      </ul>
```

Comment:
- [ ] As mentioned by the previous reviewer, you are expected to implement a filter on the home page. See image of the requirement below 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/226132900-0d5138be-7a2f-408b-9d28-785f665593d4.png)

- When the page loads, your webapp should show the list of categories that could be **_filtered out by some parameter, for example by the category name._** 
Kindly implement this.

Reply:
Thank you @bobb-Rob , i will make the changes and request a new review